applications/crossbar/src/modules/cb_notifications.erl

```diff
@@ -334,6 +387,27 @@ post(Context, Id, ?PREVIEW) ->
             crossbar_util:response('error', <<"Failed to process notification preview">>, Context)
     end.
 
+-spec build_customer_update_payload(cb_context:context()) -> wh_proplist().
+build_customer_update_payload(Context) ->
+    SenderId = case cb_context:account_id(Context) of
```

Comment:
Can we put this into a function `sender_account_id(Context)`?

```erlang
sender_account_id(Context) ->
    sender_account_id(Context, cb_context:account_id(Context)).

sender_account_id(Context, 'undefined') ->
    cb_context:auth_account_id(Context);
sender_account_id(_Context, AccountId) ->
    AccountId.
```

With appropriate spec, of course! :)